packages/react-examples/src/react/DetailsList/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -88,9 +115,11 @@ export class DetailsListCustomGroupHeadersExample extends React.Component<{}, {}
         <div className={classNames.headerAndFooter}>
           <div className={classNames.headerTitle}>{`Custom header for ${props.group!.name}`}</div>
           <div className={classNames.headerLinkSet}>
-            <Link className={classNames.headerLink} onClick={this._onToggleSelectGroup(props)}>
-              {props.selected ? 'Remove selection' : 'Select group'}
-            </Link>
+            {props.selectionMode != SelectionMode.none ? (
```

Comment:
Probably good to have this check explicitly handle the case where selectionMode is undefined.
```suggestion
            {props.selectionMode && props.selectionMode !== SelectionMode.none ? (
```

Reply:
The value of `SelectionMode.none` is `0`, hence the above change results in a compilation error.